### PR TITLE
Update etc/bash.bashrc

### PR DESCRIPTION
**Pull request by Freed-Wu**
https://github.com/archlinux/svntogit-packages/blob/packages/bash/repos/core-x86_64/system.bashrc#L22
source bash_completion

### CODE REVIEW

File: packages/bash/etc-bash.bashrc [hunk 1]
```diff
@@ -22,3 +22,6 @@ if [ -x @TERMUX_PREFIX@/libexec/termux/command-not-found ]; then
 		@TERMUX_PREFIX@/libexec/termux/command-not-found "$1"
 	}
 fi
+
+# https://github.com/archlinux/svntogit-packages/blob/packages/bash/repos/core-x86_64/system.bashrc#L22
```
Comment (tomty89) [inline packages/bash/etc-bash.bashrc:26]: I don't think the "citation" is necessary. Besides the line number could change.

Comment (Freed-Wu) [reply packages/bash/etc-bash.bashrc:26]: OK, I'll remove it.

**Review by 2096779623 — CHANGES_REQUESTED** [review]
Please bump revision.

**Comment by 2096779623** [comment]
Also modify the commit message to indicate which package has been modified.

**Comment by Freed-Wu** [comment]
OK, fixed.

**Review by 2096779623 — APPROVED** [review]
Thanks!

**Comment by tomty89** [comment]
Merged as d7bc2a7. Sorry I'm not very skilled at kind of git(hub) management for others.

**Pull request closed without merging**